code/modules/antagonists/clock_cult/structure/clockwork_camera.dm

```diff
@@ -28,11 +28,12 @@
 	var/mob/living/M = owner
 	var/mob/camera/ai_eye/remote/ratvar/cam = M.remote_control
 	var/target_loc = get_turf(cam)
+	var/area/AR = get_area(target_loc)
 	if(isclosedturf(target_loc))
-		to_chat(owner, "<span class='brass'>You cannot warp into dense objects.</span>")
+		to_chat(owner, "<span class='sevtug_small'>You cannot warp into dense objects.</span>")
```

Comment:
What's with the span changes?

Reply:
Nothing much, I'm trying to split up some CWC messages to use different spans because most of them use the brass span. It makes chat messages not blend in so much.